src/components/AbortionButton/AbortionButton.stories.js

```diff
@@ -0,0 +1,78 @@
+import {expect} from '@storybook/jest';
+import {waitFor, within} from '@storybook/testing-library';
+
+import {AnalyticsToolsDecorator} from 'story-utils/decorators';
+
+import AbortionButton from './AbortionButton';
+
+export default {
+  title: 'Private API / Abortion button',
+  component: AbortionButton,
+  decorators: [AnalyticsToolsDecorator],
+};
+
+export const Authenticated = {
+  args: {
+    isAuthenticated: true,
+  },
+  play: async ({canvasElement}) => {
+    const canvas = within(canvasElement);
+
+    await waitFor(async () => {
+      const abortButton = await canvas.queryByRole('button', {name: 'Uitloggen'});
+      await expect(abortButton).toBeVisible();
+    });
+  },
+};
+
+export const GovMetricEnabled = {
+  args: {
+    isAuthenticated: false,
+    analyticsToolsParams: {
```

Comment:
I've made some remarks in the decorator implementation about where to put these configuration params :)